docker/images/dev-cli/Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Creates an image for skycoin development
+FROM golang:1.10.0-stretch
+
+# Installs software
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        cmake \
+        libpcre3-dev \
+        gdbserver \
+        gdb \
+        vim \
+        ctags \
+        vim-scripts \
+        screen \
+        sudo \
+        doxygen \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* 
+
+# Installs Criterion
+RUN git clone --recurse-submodules -j8 https://github.com/simelo/Criterion && \
+    cd /go/Criterion && \
+    cmake . && \
+    make install && \
+    rm -r /go/Criterion
+
+RUN wget http://prdownloads.sourceforge.net/swig/swig-3.0.12.tar.gz \
+    && tar xzf swig-3.0.12.tar.gz \
+    && cd swig-3.0.12 \
+    && sh configure \
+    && make && make install \
+    && cd ../; rm -r swig-3.0.12*
+
+# Installs go development tools
+RUN go get -u github.com/derekparker/delve/cmd/dlv  && \
```

Comment:
Don't install the individual linters, see the Makefile:

```make
install-linters: ## Install linters
	go get -u github.com/FiloSottile/vendorcheck
	go get -u github.com/alecthomas/gometalinter
	gometalinter --vendored-linters --install
```

`gometalinter` vendors the linters itself, if you install the linters separately you can have problems when using gometalinter

Reply:
I install them beforehand because the skycoin source code is not there when the image is built. The only solution I see is to download the source code, run make install-linters and then delete the source code.

Reply:
OK, I'll do it that way, but we must be aware that this way of installing linters has been deprecated by gometalinter and it will be removed soon.

Reply:
Where does it say that?

Reply:
https://github.com/alecthomas/gometalinter/releases/tag/v2.0.0

Reply:
What shall be done then ?

Reply:
We'll remove it if they remove it. Stay consistent with our Makefile

Reply:
@gz-c this is already implemented in this `Dockerfile`, changes have been applied by @stdevMauricio